File: app/src/main/java/com/platdmit/simplecloudmanager/domain/repo/ServerLoadAveragesRepo.java
package com.platdmit.simplecloudmanager.domain.repo;

import com.platdmit.simplecloudmanager.domain.models.LoadAverage;

import java.util.List;

import androidx.lifecycle.MutableLiveData;
import io.reactivex.rxjava3.core.Observable;

public interface ServerLoadAveragesRepo {
    Observable<List<LoadAverage>> getServerLoadAverages(long id);
}

File: app/src/main/java/com/platdmit/simplecloudmanager/domain/repo/ServerLoadAveragesRepo.kt
package com.platdmit.simplecloudmanager.domain.repo

import com.platdmit.simplecloudmanager.domain.models.LoadAverage
import io.reactivex.rxjava3.core.Observable

interface ServerLoadAveragesRepo {
    fun getServerLoadAverages(id: Long): Observable<List<LoadAverage>>
}